rke/k8s_rke_system_images.go

```diff
@@ -8902,6 +8902,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled with v2.6.14
```

Comment:
This will be OOB after v2.6.13.
```suggestion
		// Enabled out of band post v2.6.13
```

Reply:
done